### PR TITLE
Fix several bugs when data not loaded

### PR DESCRIPTION
**Pull request by adehecq**

### CODE REVIEW

File: geoutils/georaster.py [hunk 1]
```diff
@@ -280,9 +280,14 @@ def data(self, new_data):
...
-        if new_data.shape != self._data.shape:
+        if self.is_loaded:
+            orig_shape = self._data.shape
+        else:
+            orig_shape = (self.count, self.height, self.width)
+
+        if new_data.shape != orig_shape:
             raise ValueError("New data must be of the same shape as\
- existing data: {}.".format(self.shape))
+ existing data: {}.".format(orig_shape))
```
Comment (erikmannerfelt) [inline geoutils/georaster.py:290]: There might be something with GitHub's visualisation, but it seems to me like this line is unindented. Is that true?

**Review by rhugonnet — APPROVED** [review]
Looks good :D

File: geoutils/georaster.py (hunk 1 above)
Comment (adehecq) [reply geoutils/georaster.py:290]: It's correct, but if I indent, the text that is printed has a large space. How do I avoid this?

Comment (adehecq) [reply geoutils/georaster.py:290]: I think I found it. Should I just add the quotes at the end and beginning of each newline?

Comment (adehecq) [reply geoutils/georaster.py]: See commit 63e5717

**Comment by adehecq** [comment]
Resolves #76

**Pull request merged**